policyengine_us/variables/gov/states/dc/tax/income/credits/ctc/dc_ctc_qualify_child.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class dc_ctc_eligible_children_count(Variable):
```

Comment:
```suggestion
class dc_ctc_eligible_child(Variable):
```
Make this a person bool instead, to avoid confusion since this can exceed the cap as is; and/or create a capped tax unit variable